### DISCUSSION TITLE
Welcome to Chegg_Skip_Extension Discussions!

### DISCUSSION THREAD

**Discussion by dalchandra** [dalchandra/Chegg_Skip_Extension#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jkmqtt** [comment]
hello sir 

chegg skip Questions .. chrone extension.

Notification sound is not working..

i am getting any notification sound . when questions coming on screen.

can you please solve this issue as soon as possible. 

thank you :)

**Reply by dalchandra** [reply]
Follow [this](https://github.com/dalchandra/Chegg_Skip_Extension/issues/12)

**Comment by Kishan9875** [comment]
Sir the dashboard has been changed .so could you please update chegg extension
